### PR TITLE
email: Use django-email-extras

### LINKED ISSUE #76: mail: Use django-email-extras
Use [django-email-extras](https://github.com/stephenmcd/django-email-extras) as the preferred mailing system.

To Do:
- [x] Use the provided template system
- [x] Add usage & details to README.md (related to PGP & templates)

### PR DESCRIPTION
**Pull request by nkprince007**
Uses the template system provided by the package.
Adds usage details to README.md

Fixes #76

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,8 +47,21 @@ Install pip for python3
 1. Login the admin panel and create entries in the Navbar Entries section as per your requirement.
 2. Load and use the `navbar` tag if required in any of your custom pages.
 
+### Setting up email templates
+1. Create templates in both text and html formats.
+2. Place them in the directory cited below.
```
Comment (ghoshbishakh) [inline README.md]: I think a 'how to send newsletters' will be helpful.

How do we use these templates?

Comment (nkprince007) [inline README.md]: These could be called in a way similar to [this](https://github.com/nkprince007/festcms/blob/mail-extras/festflow/views.py#L132,#L135)

Comment (nkprince007) [inline README.md]: Yeah, I understand, documentation isn't quite helpful here. I'll update them later.

Comment (ghoshbishakh) [inline README.md]: @nkprince007 so you want users to send mail by writing code?

Comment (nkprince007) [inline README.md]: As of now, it just works that way, we would need to manually create files in templates.
I too think it's a lame way.

I think it's better to put up some use to WYSIWYG editor with a form that automates this stuff.
But again, how do we put the `context` stuff into the mail?

Comment (ghoshbishakh) [inline README.md]: @nkprince007 too confusing. I think it was better before.

**Comment by nkprince007** [comment]
> I think it was better before. 

I don't understand. We already did use some markup for the mail, right?

**Comment by nkprince007** [comment]
We use the same template now too.

**Comment by ghoshbishakh** [comment]
@nkprince007 I don't see benifits. There is no useful abstraction here. is there?

**Comment by nkprince007** [comment]
Not much, I guess, just uses the additional package we added as dependency for `django-forms-builder`. 
Some configuration could be done for PGP Encryption. 
Uses text based templates when the user doesn't have HTML based email clients.

**Comment by ghoshbishakh** [comment]
@nkprince007 Okay. Go ahead and add a doc on 'how to setup newsletters'

**Comment by nkprince007** [comment]
Ok. Newsletters could be sent from admin section, right?
Just some doc on how to send them?

**Comment by ghoshbishakh** [comment]
@nkprince007 just specify that it can be done from admin.

**Comment by ghoshbishakh** [comment]
@nkprince007 and how to use a template

**Comment by nkprince007** [comment]
Okay, will modify the PR. I'm AFK right now

**Pull request merged**